cypress/integration/mailinglist-signup-flow.spec.js

```diff
@@ -240,6 +240,86 @@ describe(`Feature - Mailing list sign up : Tests execution date and time : ${new
 		cy.contains("Privacy Notice: Get into Teaching Information Service").should("exist");
 		cy.get(".site-footer-top").should("exist").next().should("exist");
 	});
+
+	it("Links through to feedback survey page", function () {
+		let rnum = Math.floor(Math.random() * 1000000000 + 1);
```

Comment:
Didn't you make a helper for this login step?

Reply:
No, actually  **enterFirstNameLastNameAndEmailAddress(firstName,lastName,email)** is my re-usable/helper method in which I'm passing 3 arguments. The reason why I've not included below step in this helper method is, based on test nature/requirement somewhere I'm keeping FirstName, Last Name constant and using random email. In some test I'm using random FirstName, Last Name and constant email. So based on test requirement I'm using below steps before the method.
**let rnum = Math.floor(Math.random() * 1000000000 + 1);
let firstName = `Testuser${rnum}firstname`;
let lastName = `Testuser${rnum}lastname`; 
let email = `testuser${rnum.toString()}@mail.co.uk`;**